module/VuFindTheme/tests/unit-tests/src/VuFindTest/ThemeInjectTemplateListenerTest.php

```diff
@@ -49,7 +49,7 @@ class ThemeInjectTemplateListenerTest extends \PHPUnit\Framework\TestCase
      */
     public function testNamespaceStripping()
     {
-        $l = new InjectTemplateListener();
+        $l = new InjectTemplateListener(['VuFind/']);
         $this->assertEquals(
```

Comment:
The tests should also include one that verifies that something in unlisted namespaces is not changed, such as
```
        $this->assertEquals(
            'vufindadmin/admin',
            $l->mapController(\VuFindAdmin\Controller\AdminController::class)
        );

```

Reply:
Also done.